src/classes/TestResult.js

```diff
@@ -83,6 +83,27 @@ export default class TestResult extends BubblingEventTarget {
 	 * Run the test(s)
 	 */
 	async run () {
+		// If not specified, give the test 10s to run
+		let timeout = this.test.maxTime ?? this.test.maxTimeAsync ?? this.options.timeout ?? 10000;
+
+		// Add a buffer to account for the time it takes to evaluate the test
+		timeout += 500;
```

Comment:
Unsure about this value either. Is there a way to get the same result without adding the buffer?